config/application.rb

```diff
@@ -58,6 +58,17 @@ class Application < Rails::Application
     config.eager_load_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('app', 'validators')
 
+    # :nocov:
+    if Rails.env.production?
```

Comment:
Since the only condition is the environment, why not include this in `config/environments/production.rb` ?